docker-compose.redis.yaml

```
@@ -1,13 +1,19 @@
 #ddev-generated
+volumes:
+  redis:
+    name: ddev-${DDEV_SITENAME}-redis
+    labels:
+      com.ddev.site-name: ${DDEV_SITENAME}
+
 services:
   redis:
     container_name: ddev-${DDEV_SITENAME}-redis
-    image: redis:6-bullseye
-    # These labels ensure this service is discoverable by ddev.
+    image: redis:${DDEV_REDIS_VERSION:-6}-alpine
```

What if someone just wants to use `6` or `7-bookworm` tag, let's give them this ability:

```suggestion
    image: redis:${DDEV_REDIS_VERSION:-6-alpine}
```

---

What about this one?